crmd/lrm.c

```diff
@@ -2331,9 +2351,20 @@ process_lrm_event(lrm_state_t * lrm_state, lrmd_event_data_t * op, struct recurr
     CRM_CHECK(op != NULL, return FALSE);
     CRM_CHECK(op->rsc_id != NULL, return FALSE);
 
+    rsc = lrm_state_get_rsc_info(lrm_state, op->rsc_id, 0);
+
+    if (op->op_status == PCMK_LRM_OP_DONE && op->rc == PCMK_OCF_OK &&
+        safe_str_eq(op->op_type, CRMD_ACTION_METADATA) && rsc) {
```

Comment:
Well, I think our metadata error handling depends on the place we will inject this call. My intuition says that we won't have a second chance to query it, so we have to either use existing cached value or fail start operation.